web-console/src/lib/functions/common/d3-random-bignumber/exponential.ts

```diff
@@ -0,0 +1,30 @@
+import { BigNumber } from 'bignumber.js'
+
+import { RandomBigNumberGenerationSource } from './generationSource'
+
+/**
```

Comment:
is this copied from soemwhere? if so we should add some attribution to the file and check license